pootle/apps/pootle_store/models.py

```diff
@@ -1750,27 +1752,11 @@ def increment_unsynced_unit_revision(self, update_revision):
 
         return count
 
-    def serialize(self):
-        from django.core.cache import caches
-
-        cache = caches["exports"]
-        rev = self.get_max_unit_revision()
-        path = self.pootle_path
-
-        ret = cache.get(path, version=rev)
-        if not ret:
-            storeclass = self.get_file_class()
-            store = self.convert(storeclass)
-            if hasattr(store, "updateheader"):
-                # FIXME We need those headers on import
-                # However some formats just don't support setting metadata
-                store.updateheader(add=True, X_Pootle_Path=path)
-                store.updateheader(add=True, X_Pootle_Revision=rev)
-
-            ret = str(store)
-            cache.set(path, ret, version=rev)
+    def deserialize(self, data):
+        self.update(StoreDeserializer(self).deserialize(data))
 
-        return ret
+    def serialize(self):
+        return StoreSerializer(self).serialize()
```

Comment:
because its related to `Store` serialization specifically, as opposed to eg `Project` or `TP` serialization etc.

atm, its only hooked up for import_export, but i was going to PR to pretty soon after to use it elsewhere.